lib/omise/resource.rb

```diff
@@ -31,23 +36,30 @@ def initialize(url, path, key)
     attr_reader :uri, :headers, :key
 
     def get(attributes = {})
-      @resource.get(params: attributes) { |r| Omise::Util.load_response(r) }
+      resource.get(params: attributes) { |r| load_response(r) }
     end
 
     def patch(attributes = {})
-      @resource.patch(attributes) { |r| Omise::Util.load_response(r) }
+      resource.patch(attributes) { |r| load_response(r) }
```

Comment:
Or maybe we could divide this differently:

```ruby
log(request, response)
load_response
```

(Sorry, I'm bombarding you with ideas. :sweat_smile:)